tests/test_module_shutting.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+import unittest
+import subprocess
+import signal
+from os.path import dirname, abspath
+from time import time
+
+
+def run_container_in_sub_process(command):
+    is_network_traffic_capture_started = False
+    parent_directory = str(dirname(dirname(abspath(__file__))))
+    output = str()
+    expected_result = False
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stdin=subprocess.PIPE, shell=False,
+                               cwd=parent_directory)
+    start_time = time()
+    for c in iter(lambda: process.stdout.read(1), b""):
+        if time() - start_time > 300:
+            os.kill(process.pid, signal.SIGINT)
+            break
+        sys.stdout.buffer.write(c)
+        output += c.decode("utf-8")
+        if "network_traffic_capture process started" in output and is_network_traffic_capture_started is False:
```

Comment:
if the messaging format changes, tests will start failing 
should we refactor so that messages are in variables and can be reused here?